public/app/features/alerting/unified/components/notification-policies/Filters.tsx

```diff
@@ -97,24 +100,26 @@ const NotificationPoliciesFilter = ({
           defaultValue={queryString}
         />
       </Field>
-      <Field label="Search by contact point" style={{ marginBottom: 0 }}>
-        <ContactPointSelector
-          selectProps={{
-            id: 'receiver',
-            'aria-label': 'Search by contact point',
-            onChange: (option) => {
-              setSearchParams({ contactPoint: option?.value?.name });
-            },
-            width: 28,
-            isClearable: true,
-          }}
-          selectedContactPointName={searchParams.get('contactPoint') ?? undefined}
-        />
-      </Field>
+      {contactPointsSupported && canSeeContactPoints && (
```

Comment:
Just making sure we hide the selector if you don't have access to any contact points. We were still making a call for the list, but it wasn't erroring as the endpoint just gives you an empty list. But regardless, probably best to not bother calling the endpoint. If you have no contact points, we can't show a meaningful filter